### PR TITLE
crystal-lang.org

### PR DESCRIPTION
**Pull request by gork7777**
issue: #78681

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
...singstar.com,ski.com.au,skidkaonline.ru,skidrowcrack.com,skype.com,slantmagazine.com,sltrib.com,snp.org,soulwire.co.uk,sourcefednews.com,spankbang.com,speakingtree.in,sportfm.ru,sportingnews.com,springer.com,sprintfeed.com,starbucks.com,stars-portraits.com,steampowered.com,suntimes.com,surfacemag.com,syl.ru,taiwantoday.tw,take40.com,tanisgazetesi.com.tr,teamtalk.com,technobuffalo.com,ted.com,temp-mail.ru,texterra.ru,thedailymeal.com,thefile.me,theglobaldispatch.com,thenational.ae,theringreport.com,thewindowsclub.com,timeline.com,timeslive.co.za,todoanimeshd.com,tomonews.net,tor.com,torontosun.com,total-croatia-news.com,truba.com,tunein.com,tv.yahoo.com,tvnz.co.nz,twitchy.com,uanews.org,unhearit.com,upmedia.mg,urbandictionary.com,usatoday.com,utoronto.ca,van.man,vanityfair.com,vectorstock.com,vellisa.ru,verisign.com,versia.ru,vesti-ukr.com,vgtimes.ru,vh1.com,video.infonet.vn,videoweed.es,virtex.com,visiontimes.com,vulture.com,wanda-group.com,washingtonpost.com,wegotthiscovered.com,wheels24.co.za,winnipegfreepress.com,wired.co.uk,wmagazine.com,wrc.com,wsj.com,xarold.com,xda-developers.com,xhamster.com,xhamster.one,xxxymovies.com,yeniasir.com.tr,yipit.com,zaochnik.ru,zapak.com##.share
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:8]: It's better for site's look to use `share-title` class (you can check it on desktop version)

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:8]: @gork7777

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt]: @gork7777
I mean `##.share-title`

Comment (gork7777) [reply SocialFilter/sections/specific.txt]: Oh my fault sorry, now correctly fixed

**Pull request merged**